release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1183,19 +1183,48 @@ As part of the continued deprecation of the Operator package manifest format, th
 
+* Previously, deleting a machine stuck in `Insufficient disk space on datastore` took longer than expected because there was no distinction between various vCenter task types. With this update, the machine-controller deletion procedure now checks the vCenter task type, and no longer blocks the deletion of the machine-controller. As a result, the machine-controller is deleted quickly.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918101[*BZ#1918101*])
 
-*Cloud Credential Operator*
+* Previously, scaling from zero annotations should not have requeued if the instance type was missing. Consequently, there were constant requeue and error space messages in the `MachineSet controller` logs. With this update, users can set the annotation manually if the instance type is not resolved automatically. As a result, scaling from zero for unknown instance types works so long as users manually provide the annotation. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1918910[*BZ#1918910*])
+
+* Previously, HTTP responses were not closed properly by the machine-api-termination-handler. Consequently, goroutines were leaked in `net.http` read and write loops, which led to high memory usage. This update ensures that HTTP responses are always closed properly. As a result, memory usage is now stable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1934021[*BZ#1934021*])
+
+* Previously, multiple client sets created inside of the MachineSet controller caused slow startup times, which resulted in pods failing readiness checks in some large clusters. Consequently, the MachineSet controller would get stuck in an endless loop. This update fixes the MachineSet controller so that it uses a single client. As a result, the MachineSet controller behaves as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1934216[*BZ#1934216*])
+
+* Previously, instances would take longer to boot when an upgrade was performed by the Machine Config Daemon on the first boot. Consequently, worker nodes would get stuck in restart loops, and machine healthchecks (MCHs) would remove the worker nodes because they did not properly start. With this update, MHCs no longer remove nodes that have not started correctly. Instead, MHCs only remove nodes when explicitly requested. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1939054[*BZ#1939054*])
+
+* Previously, a certificate signing request (CSR) approval was delayed for an unknown reason. Consequently, new machines appearing in the cluster during installation were not approved quick enough, which would prolong cluster installation. This update increases the cache resync period to 10 minutes, instead of 10 hours, to mitigate occasional API server unavailability in early installation phases. As a result, master machines are now approved faster so that cluster installation is no longer prolonged. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1940972[*BZ#1940972*])
```

Comment:
```suggestion
* Previously, a certificate signing request (CSR) approval was delayed for an unknown reason. Consequently, new machines appearing in the cluster during installation were not approved quickly enough, prolonging cluster installation. To mitigate occasional API server unavailability in early installation phases, this update changes the cache resync period from 10 hours to 10 minutes. As a result, control plane machines are now approved faster so that cluster installation is no longer prolonged. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1940972[*BZ#1940972*])
```